applications/asset_tracker_v2/src/modules/Kconfig.debug_module

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DEBUG_MODULE
+	bool "Debug module"
+
+if DEBUG_MODULE
+
+config DEBUG_MODULE_USE_CUSTOM_TRANSPORT
+	bool "Use custom transport when sending memfault data"
+	help
+	  If this option is disabled, Memfault firwmare SDKs own internal HTTP transport is used to
+	  send Memfault data. If enabled, the debug module will forward the data in chunks via
+	  events of type DEBUG_EVT_MEMFAULT_DATA_READY. The data should be forwarded via the
+	  configured cloud to Memfault. This is to save overhead associated with maintaining
+	  multiple connections.
+
+config DEBUG_MODULE_MEMFAULT_HEARTBEAT_INTERVAL_SEC
+	int "Memfault heartbeat interval in seconds"
+	default 1800
+	help
+	  Configuration that sets how often Memfault captures and prepares data that will be sent
+	  to Memfault cloud. Reduce this interval to get more frequent updates to Memfault.
+	  Note that data transmitted to Memfault cloud is still dependent on how often the
+	  application sends data, which can be manipulated by setting the applications real-time
+	  configurations.
+
+endif
+
+config DEBUG_MODULE_MEMFAULT_CHUNK_SIZE_MAX
+	int "Fragment size of data transmitted to memfault"
+	default 2048 if DEBUG_MODULE_USE_CUSTOM_TRANSPORT
```

Comment:
Shoult it be so high when using custom transport?

Reply:
I'm afraid it has to. Or else the cloud module will get spammed with tens of events of type `DEBUG_EVT_MEMFAULT_DATA_READY` in case of a core-dump, carrying smaller amounts of data. Increasing this option makes the debug module send less events carrying more data. This way the internal processing queue in the cloud module won't get overfilled. (if that happens we must to a reboot to avoid undefined behavior in the app),.